frontend/src/metabase/dashboard/containers/AutomaticDashboardApp.styled.tsx

```diff
@@ -0,0 +1,24 @@
+import styled from "styled-components";
+
+export const ListRoot = styled.ol`
+  margin: 0.5rem 0;
+`;
+
+export const ItemContent = styled.div`
+  display: flex;
+  align-items: center;
+`;
+
+export const ItemDescription = styled.div`
+  margin-left: auto;
+`;
```

Comment:
Should we add `box-sizing: border-box` to components replacing `Box`? `grid-styled` applies it [here](https://github.com/rebassjs/grid/blob/7ff2a42cb3138a639bd470182c9648fc22a5c523/src/Box.js#L15)

Reply:
We don't, we have global css adding `box-sizing: border-box` https://github.com/metabase/metabase/blob/d996e7e2b445d99ca74384d33a93eac7411f937f/frontend/src/metabase/css/core/box_sizing.css#L18. It's only needed when defined like `styled(Box.withComponent("aside")`